### PR TITLE
`puppet-lint` and better handling of depends

### PR DESCRIPTION
**Pull request by theseal**

### CODE REVIEW

File: manifests/passive.pp [hunk 1]
```diff
@@ -10,27 +11,27 @@
...
-    incl     => $nagios_config_file,
-    lens     => 'NagiosCfg.lns',
-    changes  => [
-      "set enable_notifications $enable_notifications",
-      "set obsess_over_services $obsess_over_services",
-      "set obsess_over_hosts $obsess_over_hosts",
-      "set ocsp_command obsessive_service_handler",
-      "set ochp_command obsessive_host_handler"
+  nagioscfg::command { 'obsessive_host_handler':
+    command_line => "/usr/share/icinga/plugins/eventhandlers/distributed-monitoring/send_nsca_host_or_service_check_result '${nsca_server}' '/etc/send_nsca.cfg' '\$HOSTNAME\$' '\$HOSTSTATE\$' '\$HOSTOUTPUT\$\\n\$LONGHOSTOUTPUT\$|\$HOSTPERFDATA\$'" # lint:ignore:140chars
```
Comment (mickenordin) [inline manifests/passive.pp]: Should we use string interpolation to get this below 140 chars?

Something like: 
```
$plugin_path = '/usr/share/icinga/plugins/eventhandlers/distributed-monitoring/send_nsca_host_or_service_check_result'
...
   command_line => "${plugin_path} '${nsca_server}' '/etc/send_nsca.cfg' '\$HOSTNAME\$' '\$HOSTSTATE\$' '\$HOSTOUTPUT\$\\n\$LONGHOSTOUTPUT\$|\$HOSTPERFDATA\$'"
...
```

File: manifests/passive.pp [hunk 2]
```diff
@@ -10,27 +11,27 @@
...
+  }
+  -> nagioscfg::command { 'obsessive_service_handler':
+    command_line => "/usr/share/icinga/plugins/eventhandlers/distributed-monitoring/send_nsca_host_or_service_check_result '${nsca_server}' '/etc/send_nsca.cfg' '\$HOSTNAME\$' '\$SERVICEDESC\$' '\$SERVICESTATE\$' '\$SERVICEOUTPUT\$\\n\$LONGSERVICEOUTPUT\$|\$SERVICEPERFDATA\$'" # lint:ignore:140chars
```
Comment (mickenordin) [inline manifests/passive.pp]: Same here?

File: manifests/pushover.pp [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
-   }
+  file { '/usr/lib/nagios/plugins/notify_by_pushover':
+    ensure  => file,
+    owner   => root,
+    group   => root,
+    mode    => '0755',
+    content => template('nagioscfg/notify_by_pushover.erb')
+  }
+  nagioscfg::command { 'notify-host-by-pushover':
+  command_line => "/usr/lib/nagios/plugins/notify_by_pushover -u '\$CONTACTADDRESS1\$' -a '\$CONTACTADDRESS2\$' -t '\$NOTIFICATIONTYPE\$ Host Alert: \$HOSTNAME\$ is \$HOSTSTATE\$' -m 'Notification Type: \$NOTIFICATIONTYPE\$|Host: \$HOSTNAME\$|State: \$HOSTSTATE\$|Address: \$HOSTADDRESS\$|Info: \$HOSTOUTPUT\$||Date/Time: \$LONGDATETIME\$'" # lint:ignore:140chars
```
Comment (mickenordin) [inline manifests/pushover.pp]: And here :)

File: manifests/pushover.pp [hunk 4]
```diff
@@ -1,15 +1,16 @@
...
+  }
+  nagioscfg::command { 'notify-service-by-pushover':
+    command_line => "/usr/lib/nagios/plugins/notify_by_pushover -u '\$CONTACTADDRESS1\$' -a '\$CONTACTADDRESS2\$' -t '\$NOTIFICATIONTYPE\$ Service Alert: \$HOSTALIAS\$/\$SERVICEDESC\$ is \$SERVICESTATE\$' -m 'Notification Type: \$NOTIFICATIONTYPE\$||Service: \$SERVICEDESC\$|Host: \$HOSTALIAS\$|Address: \$HOSTADDRESS\$|State: \$SERVICESTATE\$||Date/Time: \$LONGDATETIME\$||Additional Info:||\$SERVICEOUTPUT\$'" # lint:ignore:140chars
```
Comment (mickenordin) [inline manifests/pushover.pp]: Also here :)

**Review by mickenordin — APPROVED** [review]
This looks good, I left a comment about lenght of lines, which could be seen as an optional improvement maybe.

**Comment by theseal** [comment]
Updated according to your commends. The manifest in whole seems to pass on my test server but I don't have any servers running pushover or the passive stuff. Do you?

**Review by mickenordin — APPROVED** [review]
Looks good to me!

**Comment by mickenordin** [comment]
> Updated according to your commends. The manifest in whole seems to pass on my test server but I don't have any servers running pushover or the passive stuff. Do you?

I don't, but I still think it looks safe to merge, so I will do that :)

**Pull request merged**